public/components/event_analytics/explorer/timechart_header/timechart_header.tsx

```diff
@@ -54,7 +60,10 @@ export function TimechartHeader({
             })}
             delay="long"
           >
-            <EuiText data-test-subj="discoverIntervalDateRange" size="s"></EuiText>
+            <EuiText data-test-subj="discoverIntervalDateRange" size="s">
+              {reformatDate(datemath.parse(startTime ?? DEFAULT_DATETIME_STRING))} -{' '}
+              {reformatDate(datemath.parse(endTime ?? DEFAULT_DATETIME_STRING))}
```

Comment:
We can edit this to a single line: `{reformatDate(datemath.parse(startTime ?? DEFAULT_DATETIME_STRING)) - reformatDate(datemath.parse(endTime ?? DEFAULT_DATETIME_STRING))}`

Also, will the default time be `now-now` if start and end time are not passed?

Reply:
The default time (which may not show up since the whole component gets loaded at once) will actually show up as a specific time being that exact moment, since datemath.parse() turns strings such as "now" into datetime strings.